src/test/java/com/wikia/webdriver/testcases/mobilewikitests/ArticlePageTests.java

```diff
@@ -135,25 +135,25 @@ public void mercury_article_navigateToArticlesWithColonAndQuestionMark() {
     //navigation.openPageLink(5);
 
     Assertion.assertFalse(driver.getCurrentUrl().contains(encodedColonUrl));
-    Assertion.assertTrue(MercurySubpages.COLON.toLowerCase().contains(article.getArticleTitle().toLowerCase()));
+    Assertion.assertTrue(MobileSubpages.COLON.toLowerCase().contains(article.getArticleTitle().toLowerCase()));
 
     globalNavigationMobile.openNavigation();
     //navigation.openSubMenu(3);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
do we need this commented code?

Reply:
we need that commented line until I rewrite that test using new local nav